### PR TITLE
Bump disinfectant_power of cattail slightly to 2

### PR DESCRIPTION
**Pull request by esarbe**
https://www.tsijournals.com/articles/phytochemical-screening-and-antimicrobial-investigation-of-typha-angustifolia-linn.pdf

To quote
> The result showed that methanolic extract of Typha angustifolia have potent antifungal and
antibacterial activities.

Given the  - admittedly somewhat sparse - research and the long history of cattail in medicine I suggest to re-balance cattail jelly to disinfectant_power to 2, which seems to be a fair compromise as it leaves some early use for the plant.

#### Summary
Category "Brief description"

With a disinfectant_power power of 1, cattail jelly is now pretty useless. This is a shame since it provides a nice survivalist favor for the early game.

#### Purpose of change

Have some application for cattail jelly, which would otherwise pretty useless.

#### Describe the solution

Bump disinfectant_power of cattail jelly up to 2.

#### Describe alternatives you've considered

- removing cattail jelly altogether
- bumping disinfectant_power back up to 3

#### Testing

none, admittedly

#### Additional context

https://www.tsijournals.com/articles/phytochemical-screening-and-antimicrobial-investigation-of-typha-angustifolia-linn.pdf

To quote
> The result showed that methanolic extract of Typha angustifolia have potent antifungal and
antibacterial activities.

### CODE REVIEW

**Comment by Venera3** [comment]
1) You're not doing methanol extraction or any separation on the gunk
2) This is in vitro "data" of isolated compounds (quotation marks for not actually including inhibition distances or MIC).

**Comment by anothersimulacrum** [comment]
For the future: #54790

**Comment by kevingranade** [comment]
It's odd that the paper did a qualitative test and did not report the results, instead reporting an evaluation of the results.

Luckily we do have a paper that was more straightforward: https://www.researchgate.net/publication/256839414_Phytochemical_screening_and_in_vitro_antimicrobial_activity_of_Typha_angustifolia_Linn_leaves_extract_against_pathogenic_gram_negative_micro_organisms

It reports a MIC of 750 ug/mL for some bacteria,  and a significantly higher (worse) number for others.

Putting that in context, we have a study using triple-antibiotic https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3467137/
Triple antibiotic MIC is around 77.5 ug/mL against a particular target, so take that as the standard we hope to approach.

This suggests that the cattail extract requiring 10x the concentration is dramatically less efficacious than triple antibiotic. 

On top of that, without chemical extraction the concentration of the active ingredient is definitely lower than 750 ug/mL, meaning it's hardly going to be acting as a bacteriocidal at all.

In conclusion, having a diseinfectant_power of 1 for cattail jelly is a very generous compromise already.
